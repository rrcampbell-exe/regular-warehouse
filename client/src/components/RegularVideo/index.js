import React from 'react';

const RegularVideo = () => {
  return (
    <video id="regular-video" controls>
      <source src="./video/irregular-warehouse.mp4"></source>
    </video>
  )
}

export default RegularVideo;