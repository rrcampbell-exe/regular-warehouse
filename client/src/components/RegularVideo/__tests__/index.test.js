import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import RegularVideo from '..';

afterEach(cleanup);

describe('RegularVideo component', () => {
    it('renders', () => { // this test passes
        render(<RegularVideo/>)
    })
})