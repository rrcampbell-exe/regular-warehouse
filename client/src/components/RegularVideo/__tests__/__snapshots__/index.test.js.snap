// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegularVideo component matches snapshot 1`] = `
<DocumentFragment>
  <video
    class="regular-video-invisible"
    id="regular-video"
  >
    <source
      src="./video/irregular-warehouse.mp4"
    />
  </video>
</DocumentFragment>
`;
